InstagramKit-Example/InstagramKit-Example/Classes/View/IKCollectionViewController.m

```diff
@@ -74,11 +74,12 @@ - (void)loadMedia
     [self.collectionView reloadData];
     
     if (isSessionValid) {
-        [self requestSelfFeed];
+        [self requestSelfRecentMedia];
     }
     else
     {
-        [self requestPopularMedia];
+    // Not support with new API
```

Comment:
Could you please remove the entire else block instead of leaving comments in there?